### PR TITLE
add petting zoo type

### LINKED ISSUE #6238: Petting zoo shouldn't have a giraffe icon
from OSM World discord:

>Casper:
Regarding the icons, maybe petting zoos shouldn't have a giraffe icon. Example: https://www.openstreetmap.org/way/852058333

>Jasper:
probably rather be a sheep or rabbit

it's true, i haven't seen many petting zoos where you can pet a giraffe 😅

### PR DESCRIPTION
**Pull request by RedAuburn**
fixes #6238 

icon:

(icon source has been added to copyright.html)

### CODE REVIEW

**Review by vng — APPROVED** [review]

File: data/copyright.html [hunk 1]
```diff
@@ -335,6 +335,9 @@
 
   <li><a href="https://github.com/mapbox/maki" title="Maki">Maki</a><br>
     &copy; 2022 Mapbox <a href="#cc0-1" class="license">CC0 1.0 Universal</a></li>
+  
+  <li><a href="https://www.onlinewebfonts.com/svg" title="OnlineWebFonts">OnlineWebFonts</a><br>
```
Comment (biodranik) [inline data/copyright.html:339]: This URL shows 404. Where did you find their license?

**Review by biodranik — CHANGES_REQUESTED** [review]

File: data/copyright.html (hunk 1 above)
Comment (RedAuburn) [reply data/copyright.html:339]: ah oops, it should be https://www.onlinewebfonts.com/icon/

 https://www.onlinewebfonts.com/icon/74710

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -29109,6 +29109,48 @@
...
+    ko = 페팅 동물원
+    lt = Petting Zoo
+    mr = पेटिंग प्राणीसंग्रहालय
+    nb = Dyrehage
+    nl = Kinderboerderij
+    pl = Małe Zoo
+    pt = Jardim Zoológico
+    pt-BR = Jardim Zoológico
+    ro = Grădina Zoologică
+    ru = Зоопарк
```
Comment (pastk) [inline data/strings/types_strings.txt:29142]: Контактный зоопарк

File: data/strings/types_strings.txt [hunk 3]
```diff
@@ -29109,6 +29109,48 @@
...
+    sk = Zoologická záhrada s domácimi zvieratami
+    sv = Petting Zoo
+    sw = Kufuga Zoo
+    th = สวนสัตว์ลูบคลำ
+    tr = Sevişme Hayvanat Bahçesi
+    uk = Зоопарк для домашніх тварин
```
Comment (pastk) [inline data/strings/types_strings.txt:29148]: Контактний зоопарк

File: data/styles/clear/include/priorities_4_overlays.prio.txt [hunk 4]
```diff
@@ -789,6 +789,7 @@ tourism-resort                                      # icon z16- (also has captio
 === 2750
 
 tourism-motel                                       # icon z16- (also has caption(optional) z16-)
+tourism-zoo-petting                                 # icon z15- and caption z13-14 (also has caption(optional) z15-)
```
Comment (pastk) [inline data/styles/clear/include/priorities_4_overlays.prio.txt:792]: please remove iconless captions from z13-14

Comment (pastk) [reply data/styles/clear/include/priorities_4_overlays.prio.txt:792]: also its quite an attraction for local people, isn't it?
I think put it next to `amenity-pharmacy` or `amenity-fountain`

**Review by pastk — CHANGES_REQUESTED** [review]

File: data/copyright.html (hunk 1 above)
Comment (biodranik) [reply data/copyright.html:339]: Sorry, I still don't see the license text in any of the links.

Comment (RedAuburn) [reply data/copyright.html:339]: it is a bit hidden, but when you press download it shows the attribution info, and the license is specified.
image

File: data/styles/clear/include/priorities_4_overlays.prio.txt (hunk 4 above)
Comment (RedAuburn) [reply data/styles/clear/include/priorities_4_overlays.prio.txt]: fixed :)

**Comment by RedAuburn** [comment]
@pastk PTAL, should be good now

File: data/copyright.html (hunk 1 above)
Comment (pastk) [reply data/copyright.html]: @biodranik is it good to be merged?

**Review by biodranik — APPROVED** [review]

**Comment by biodranik** [comment]
A rebase is needed.

**Comment by RedAuburn** [comment]
> A rebase is needed.

done

**Pull request merged**